tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/populate_survey_conduct_ext_test.py

```diff
@@ -0,0 +1,111 @@
+"""
+Integration test for PopulateSurveyConductExt.
+"""
+
+# Python Imports
+import os
+
+# Third party imports
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.populate_survey_conduct_ext import PopulateSurveyConductExt
+from common import EXT_SUFFIX, JINJA_ENV, QUESTIONNAIRE_RESPONSE_ADDITIONAL_INFO, SURVEY_CONDUCT
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+INSERT_SURVEY_CONDUCT_EXT = JINJA_ENV.from_string("""
+    INSERT INTO `{{project}}.{{dataset}}.survey_conduct_ext` 
+        (survey_conduct_id, src_id, language)
+    VALUES
+        (11, 'pi/pm', NULL),
+        (12, 'site foo', NULL),
+        (13, 'site foo', NULL),
+        (14, 'site bar', NULL)
+""")
+
+INSERT_QUESTIONNAIRE_RESPONSE_ADDITIONAL_INFO = JINJA_ENV.from_string("""
+    INSERT INTO `{{project}}.{{dataset}}.questionnaire_response_additional_info` 
+        (questionnaire_response_id, type, value)
+    VALUES
+        (11, 'LANGUAGE', 'en'),
+        (11, 'NON_PARTICIPANT_AUTHOR_INDICATOR', 'CATI'),
+        (11, 'CODE', 'TheBasics'),
+        (12, 'LANGUAGE', 'es'),
+        (13, 'NON_PARTICIPANT_AUTHOR_INDICATOR', 'CATI'),
+        (14, 'CODE', 'TheBasics')
+""")
+
+
+class PopulateSurveyConductExtTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # set the test project identifier
+        project_id = os.environ.get(PROJECT_ID)
+        cls.project_id = project_id
+
+        # set the expected test datasets
+        dataset_id = os.environ.get('RDR_DATASET_ID')
+        cls.dataset_id = dataset_id
+        sandbox_id = dataset_id + '_sandbox'
+        cls.sandbox_id = sandbox_id
```

Comment:
I've reviewed some other PRs recently that have collapsed lines 50 and 51 into one like so:
`cls.project_id = os.environ.get(PROJECT_ID)`

Same for lines 54 and 55:
`cls.dataset_id = os.environ.get('RDR_DATASET_ID')`

And also removed string concatenation in favor of f-strings on line 56.

For consistency, could you make those few changes?